### PR TITLE
sla: add initial sla wording

### PR DESCRIPTION
**Pull request by euank**
cc @ilianaw @LinuxMercedes; I'm not really happy with the words here, but I want something to link to regarding nine-fives.

If you have suggestions for how to spice this up or make it read better, I'd appreciate them!

### CODE REVIEW

File: sla.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+In reality, this is all just a pun on the more common [five nines](https://en.wikipedia.org/wiki/Five_nines), and we will strive to do far better than we promise.
+
+That being said, we are running this as a hobby. We'll try our darndest to keep
+things running well, but we don't have 24/7 oncall nor a stack of spare
+equipment for every situation.
+
+## Breaking the SLA
+
+If the SLA is broken, we will refund you the full cost of the service for the month in question, plus an additional 55 cents (plus or minus taxes).
```
Comment (LinuxMercedes) [inline sla.md]: "plus an additional nine-fives percent of a dollar"

File: sla.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+## What it is
+
+The Wobscale nine-fives SLA promises that a given service will be available
+at least 55.5555555% of each month.
+
+In reality, this is all just a pun on the more common [five nines](https://en.wikipedia.org/wiki/Five_nines), and we will strive to do far better than we promise.
+
+That being said, we are running this as a hobby. We'll try our darndest to keep
+things running well, but we don't have 24/7 oncall nor a stack of spare
+equipment for every situation.
```
Comment (ilianaw) [inline sla.md]: "We can get a lot of replacement equipment on Prime though."

File: sla.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## Breaking the SLA
+
+If the SLA is broken, we will refund you the full cost of the service for the month in question, plus an additional 55 cents (plus or minus taxes).
+
+Terms and conditions may apply and this portion of the agreement may change without notice, even retroactively.
+
+## What it covers
```
Comment (ilianaw) [inline sla.md]: It probably shouldn't cover pre-announced downtime (e.g. our upcoming switch to Wob Net 2 whenever I finally get that shit working)

Comment (euank) [reply sla.md]: I think it's such a generous number it doesn't matter much either way.

If we do manage to have planned downtime long enough to even approach this, we've done something quite wrong and should be accountable regardless.

**Comment by euank** [comment]
Note, it's not actually linked anywhere yet. We can fix that at some point, I just thought it was worth writing down somewhere more formal than as a perennial meme on irc.

**Pull request merged**